docs/guide/getstarted/configuration.md

```diff
@@ -293,3 +293,6 @@ Similarly for mocha:
         requires: ['./test/helpers/common.js']
     },
 ```
+##specFileRetries
```

Comment:
This should be moved up to the other options like `connectionRetryTimeout` and should have a similar format than these

Reply:
Thanks, I did sign the CLA as far as I can tell.  I will fix these other issues asap.
https://cla.js.foundation/webdriverio/webdriverio?pullRequest=2614
->
"You have signed the CLA for webdriverio/webdriverio"